nuitka/tools/release/Documentation.py

```diff
@@ -28,68 +28,65 @@
 from nuitka.utils.FileOperations import getFileList
 
 
+def which(prog):
+    """
+    cross version program detection.
+
+    Whenever Nuitka decides to drop Python versions older than 3.3
+    the code in the else block can be removed.
+    """
+    def real(f):
+        def wrap(*args):
+            cmd = ""
+            if sys.version_info > (3, 3):
```

Comment:
That is probably not a good way to compare. You probably mean `>=` here.

Reply:
Yes, You are correct here.